build-stanza.proj

```diff
@@ -1,5 +1,4 @@
 include "core/stanza.proj"
-include "core/debug/stanza.proj"
```

Comment:
Decided to move files out from `core/debug` and to top-level `debug` folder.

By default, `core/stanza.proj` source code is deliberately not reachable via the root `stanza.proj` so that users do not accidentally compile the core packages from source.